server/repositories/__tests__/cmsApi.spec.js

```diff
@@ -24,58 +24,63 @@ describe('CmsApi', () => {
     nock.restore();
   });
 
-  describe('primaryMenu', () => {
-    const path =
-      '/jsonapi/node/landing_page?fields[node--landing_page]=title,field_moj_description,drupal_internal__nid,field_moj_prisons';
+  describe('getTopics', () => {
+    const path = `/jsonapi/prison/berwyn/taxonomy_term?${topicsQuery}`;
 
-    it('should handle no items', async () => {
-      mockDrupal.get(path).reply(200, primaryMenuResponse([]));
+    it('should handle returning no items', async () => {
+      mockDrupal.get(path).reply(200, jsonApiResponse([]));
 
-      const response = await cmsApi.primaryMenu(792);
+      const response = await cmsApi.getTopics('berwyn');
 
       expect(response).toStrictEqual([]);
     });
 
     it('should handle empty responses', async () => {
       mockDrupal.get(path).reply(200);
 
-      const response = await cmsApi.primaryMenu(792);
+      const response = await cmsApi.getTopics('berwyn');
 
       expect(response).toStrictEqual([]);
     });
 
     it('should propagate errors', () => {
       mockDrupal.get(path).reply(500, 'unexpected error');
 
-      return expect(cmsApi.primaryMenu(792)).rejects.toEqual(
+      return expect(cmsApi.getTopics('berwyn')).rejects.toEqual(
         Error('Request failed with status code 500'),
       );
     });
 
+    it('propogates 404 as error', () => {
+      mockDrupal.get(path).reply(404, 'unexpected error');
+
+      return expect(cmsApi.getTopics('berwyn')).rejects.toEqual(
+        Error('Request failed with status code 404'),
+      );
+    });
+
     it('should format and return matching item', async () => {
       mockDrupal.get(path).reply(
         200,
-        primaryMenuResponse([
-          landingPage({
+        jsonApiResponse([
+          tag({
```

Comment:
Moved to use drupal domain terminology within the api/data layer: `tag` `categories` 

and exposing app domain terminology throughout the rest of the codebase: `topics`